dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -788,56 +523,6 @@ INP_DLLEXPORT void dpnp_prod_c(void *result_out,
                                const void *initial,
                                const long *where);
 
-/**
- * @ingroup BACKEND_API
- * @brief Range of values (maximum - minimum) along an axis.
- *
- * @param [in]  q_ref               Reference to SYCL queue.
- * @param [out] result_out          Output array.
- * @param [in]  result_size         Size of output array.
- * @param [in]  result_ndim         Number of output array dimensions.
- * @param [in]  result_shape        Shape of output array.
- * @param [in]  result_strides      Strides of output array.
- * @param [in]  input_in            First input array.
- * @param [in]  input_size          Size of first input array.
- * @param [in]  input_ndim          Number of first input array dimensions.
- * @param [in]  input_shape         Shape of first input array.
- * @param [in]  input_strides       Strides of first input array.
- * @param [in]  axis                Axis.
- * @param [in]  naxis               Number of elements in axis.
- * @param [in]  dep_event_vec_ref   Reference to vector of SYCL events.
- */
-template <typename _DataType>
-INP_DLLEXPORT DPCTLSyclEventRef
-    dpnp_ptp_c(DPCTLSyclQueueRef q_ref,
```

Comment:
Can we remove `DPNPFuncName::DPNP_FN_SUBTRACT_EXT` since there is no `dpnp_ptp_c` function?